Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
-git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+git_source(:github) { |repo| 'https://github.com/#{repo}.git' }
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `repo`. You can omit the argument if you don't care about it.